refinery/ui/source/js/data-set-import/controllers/file-upload-s3.spec.js

```diff
@@ -0,0 +1,269 @@
+(function () {
+  'use strict';
+
+  describe('RefineryFileUploadS3Ctrl', function () {
+    var $controller;
+    var $rootScope;
+    var s3UploadService;
+    var ctrl;
+    var scope;
+
+    beforeEach(module('refineryApp'));
+    beforeEach(module('refineryDataSetImport'));
+    beforeEach(inject(function (_$controller_, _$rootScope_, _s3UploadService_) {
```

Comment:
At some point we agreed to update this syntax, since it wasn't necessary.  No need to wrap dependencies.